packages/core-components/src/layout/HeaderLabel/HeaderLabel.tsx

```diff
@@ -64,9 +60,11 @@ export const HeaderLabel = ({ label, value, url }: HeaderLabelProps) => {
     />
   );
   return (
-    <span className={classes.root}>
-      <Typography className={classes.label}>{label}</Typography>
-      {url ? <Link href={url}>{content}</Link> : content}
-    </span>
+    <Grid item>
```

Comment:
Thanks for questioning it! My reasoning is, that the `HeaderLabel` is specifically only used inside of the `Header` & therefor will always be rendered inside of the `Grid` container. Moving the `Grid` component inside of the `HeaderLabel` makes the JSX better readable. As it was only a `span` before I don't think there is a use case yet to pass down/ modify the Grid layout. If this would be the case, the `props` could be passed through to the `Grid` component.

Reply:
Makes sense!

Reply:
Awesome! :) So you would approve this PR as well? From my site it's ready to :shipit: :)

Reply:
@timbonicus I think you wanted to take a quick look before merging?